packages/hash/frontend/src/components/GlideGlid/utils/use-grid-tooltip/grid-tooltip-manager.ts

```diff
@@ -94,20 +85,27 @@ export class GridTooltipManager {
 
       const actualTooltipX = tooltipX - rect.x;
 
-      if (hoverX > actualTooltipX && hoverX < actualTooltipX + iconSize) {
-        tooltipCount++;
-
-        showTooltip({
-          text: tooltip.text,
-          iconX: actualTooltipX + iconSize / 2,
-          col,
-          row,
-        });
-      }
+      const interactable = InteractableManager.create(args, {
+        id: i,
```

Comment:
Changed the `i` to `tooltip-${i}` to prevent possible clashes. About combining these two, check my comment below. https://github.com/hashintel/hash/pull/1386#issuecomment-1314768356